File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] arr = new int[30];
        for(int i = 1; i <= 28; i++){
            arr[in.nextInt()-1]++;
        }
        for(int i = 0; i < 30; i++){
            if(arr[i] == 0)
                System.out.println(i+1);
        }
    }
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    val arr = IntArray(30,{0})
    for(i in 0..27){
        arr[sc.nextInt()-1]++;
    }
    for(i in 0..29)
        if(arr[i] == 0)
            println(i+1)
}